nlmod/mdims/resample.py

```diff
@@ -582,16 +511,15 @@ def fillnan_dataarray_structured_grid(xar_in):
     arr_out = values_out.reshape(xar_in.shape)
 
     # bathymetry without nan values
```

Comment:
lots of mentions of bathymetry in this generic function

Reply:
fixed